internal/tiger/cmd/mcp_install.go

```diff
@@ -162,8 +182,21 @@ func getValidEditorNames() []string {
 	return validNames
 }
 
-// installMCPForClient installs the Tiger MCP server configuration for the specified client
-func installMCPForClient(clientName string, createBackup bool, customConfigPath string) error {
+// InstallMCPForClient installs an MCP server configuration for the specified client.
+// This is a generic, configurable function exported for use by external projects via pkg/mcpinstall.
+// Required options: ServerName, Command, Args must all be provided.
+func InstallMCPForClient(clientName string, opts InstallOptions) error {
```

Comment:
Any reason not to include `ClientName` in the `InstallOptions` struct, instead of it being a separate parameter? Especially since there are already other required fields in `InstallOptions`.